benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md

```diff
@@ -0,0 +1,31 @@
+# To run a workload with custom parameters through JupyterLab terminal, follow these steps:
+
+1. Open the terminal
+
+```
+cd /notebooks/templates/run_workloads
+```
+
+2. Clone the repository if not already cloned: 
+
+```
+git clone https://github.com/redhat-performance/benchmark-runner
+or 
+update the repository with "git pull" if already cloned.
+```
+
+3. Change the directory to benchmark-runner:
+
+```
+cd benchmark-runner
+```
+
+4. **MUST**: Update the necessary environment variables in benchmark_runner/main/environment_variables.py, e.g., ('WORKLOAD', 'vdbench_vm').
+
+5. For custom workload parameters, configure the templates in benchmark_runner/common/template_operations/templates.
+
+6. Run the following command in the terminal:
+
+```
+PYTHONPATH=. python3 benchmark_runner/main/main.py "$@"
```

Comment:
The `"$@"` is used for processing command line arguments.

Why did you delete run_workload.sh?  It's simpler to run `run_workload.sh` than to issue a python command line with PYTHONPATH set.

Reply:
You right, see the updated code